### ISSUE TITLE
Homebridge icon does not redirect rigth

### ISSUE THREAD

**Issue by alexisteleco** [oznu/homebridge-syno-spk#21]
I have already installed the package, and everything is working just right, there is only one thing that is a little strange: 
The icon placed in my Synology NAS (DS916+ with DSM 6.2.1-23824 Update 6 [latest at this time]) is redirecting me to myhostname:8581, but the Homebridge-UI-X is running under port 8080. 
If I try to reach port 8080, it is reachable and it is working as expected (even with certs and https), but the icon is linking to port 8581, where the service itself is running. 

Do you know why this is happening and how to solve it?

**Comment by oznu** [comment]
Hi @alexisteleco,

The link on the DSM desktop is static and expects the default port to be used (8581). I did attempt to make it dynamically update but had little success.

You can edit this file to change the port and scheme:

```
/volume1/@appstore/HomebridgeDocker/ui/config
```

**Comment by alexisteleco** [comment]
I tried what you said and it worked perfectly!
As a side-note, I had to restart the "Homebridge app" and not only the docker container.
<img width="387" alt="image" src="https://user-images.githubusercontent.com/17027081/56869486-c2f72880-6a01-11e9-9ba2-cf9c9cfd064e.png">
Thanks a lot for the info!
Your work is highly appreciated ;) 
Also, a note for other users: is not necessary to change this file in order to change the scheme, it can be done as mentioned in the Homebridge webpage.
